gamemodes/terrortown/gamemode/client/cl_help.lua

```diff
@@ -339,71 +336,27 @@ function HELPSCRN:ShowSubmenu(menuClass)
 
 	-- BUILD GENERAL BOX STRUCTURE
 	local navArea = vgui.Create("DNavPanelTTT2", frame)
-	navArea:SetSize(widthNav, heightNav - vskin.GetHeaderHeight() - vskin.GetBorderSize())
+	navArea:SetSize(widthNav)
```

Comment:
```suggestion
	navArea:SetWide(widthNav)
```

So it is a bit more readable